microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/core/java/codeaction/InsertAnnotationAttributeQuickFix.java

```diff
@@ -77,13 +80,22 @@ public CodeAction resolveCodeAction(JavaCodeActionResolveContext context) {
 		ChangeCorrectionProposal proposal = new InsertAnnotationAttributeProposal(name, context.getCompilationUnit(),
 				annotation, 0, attributeName);
 		try {
-			toResolve.setEdit(context.convertToWorkspaceEdit(proposal));
+			WorkspaceEdit we = context.convertToWorkspaceEdit(proposal);

```

Comment:
Why you need to do that?

Reply:
The reason I did this was so that the two `CodeActionFactory.java` classes are the same. The `microprofile.jdt` one was making `edit` an empty map, and the `microprofile.ls` one was making `edit` `null`. I think it makes sense to be consistent about which one CodeActions have, so I chose to set it to `null` everywhere. Unfortunately, the jdt.ls utils all set `edit` to an empty map, so I remove it here.

Reply:
What is the problem if changes is an empty map?

Reply:
There is no difference in behaviour between an empty map and a null entry, but the tests will fail if there is an empty map and a null entry is expected or vice versa.